test/e2e/save_test.go

```diff
@@ -188,6 +188,7 @@ var _ = Describe("Podman save", func() {
 
 		cmd = exec.Command("cp", "sign/key.gpg", "/tmp/key.gpg")
 		Expect(cmd.Run()).To(Succeed())
+		defer os.Remove("/tmp/key.gpg")
```

Comment:
Should that also be under cachedir?

Reply:
We need to fix the `keyPath` value from `/tmp/key.gpg` to `cachePath("")/key.gpg`  in `e2e/sign/policy.json`.

https://github.com/containers/podman/blob/a702d442e98850f56bc86ee4d6adf5f493a06201/test/e2e/sign/policy.json#L13

I haven't come up with a best way, so I remove the `key.gpg` by `defer`.